config/default/manager_auth_proxy_patch.yaml

```diff
@@ -10,9 +10,10 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
+        image: quay.io/brancz/kube-rbac-proxy:v0.15.0
         args:
         - "--secure-listen-address=0.0.0.0:8443"
+        - --http2-disable
```

Comment:
Please align with other args and add quotes.
Also the PR title is confusing, sounds like updating a var?